content/id/docs/tasks/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Tugas-tugas (_Tasks_)
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (_Dashboard_)
+
+Menyebarkan dan mengakses Dashboard antarmuka pengguna berbasis web untuk 
+membantu kamu mengelola dan memantau aplikasi yang dimasukkan ke dalam kontainer
+di Kubernetes.
+
+## Menggunakan Baris Perintah kubectl
+
+Instalasi dan konfigurasi utilitas baris perintah `kubectl` yang digunakan untuk
+mengelola secara langsung klaster Kubernetes.
+
+## Mengkonfigurasi Pod and Kontainer
```

Comment:
```suggestion
## Mengkonfigurasi Pod and Container
```

Reply:
I will keep `Kontainer` because most of the other translations already use that.

Reply:
we use both Kontainer and Container (K8s Object)

this subtitle refer to K8s Object

Reply:
@ariscahyadi